vttools/to_wrap/tomo.py

```diff
@@ -0,0 +1,533 @@
+    ----------
+    data : np.ndarray
+        updated projection data
+    """
+
+    # create xtomo_dataset object and initialize it
+    d = _generate_xtomo_object(data=data, theta=theta, center=center)
+
+    data = d.phase_retrieval(overwrite=False)
+
+    return data
+
+def gridrec(data, theta, center):
+    """GridRec reconstruction
+
+    Parameters
+    ----------
+    data : np.ndarray
+        input projections of the sample
+
+    theta : np.ndarray
+        the angle list of projections
+
+    center : float
+        the center of projections
+
+    Returns
+    ----------
+    data_recon : np.ndarray
+        the reconstructed data by GridRec algorithm
+    """
+
+    # create xtomo_dataset object and initialize it
+    d = _generate_xtomo_object(data=data, theta=theta, center=center)
+
+    data_recon = d.gridrec(overwrite=False)
+
+    return data_recon
+
+def sirt(data, white, dark, theta, center, iters=1):
+    """SIRT reconstruction
+
+    Parameters
+    ----------
+    data : np.ndarray
+        input projections of the sample
+
+    white : np.ndarray
+        white background
+
+    dark : np.ndarray
+        dark background
+
+    theta : np.ndarray
+        the angle list of projections
+
+    center : float
+        the center of projections
+
+    iters : int, optional
+        the number of iterations
+
+    Returns
+    ----------
+    data_recon : np.ndarray
+        the reconstructed data by SIRT algorithm
+    """
+
+    # create xtomo_dataset object and initialize it
+    d = _generate_xtomo_object(data=data, white=white, dark=dark, theta=theta, center=center)
+
+    data_recon = d.sirt(iters=iters, overwrite=False)
+
+    return data_recon
+
+def art(data, white, dark, theta, center, iters=1):
+    """ART reconstruction
+
+    Parameters
+    ----------
+    data : np.ndarray
+        input projections of the sample
+
+    white : np.ndarray
+        white background
+
+    dark : np.ndarray
+        dark background
+
+    theta : np.ndarray
+        the angle list of projections
+
+    center : float
+        the center of projections
+
+    iters : int, optional
+        the number of iterations
+
+    Returns
+    ----------
+    data_recon : np.ndarray
+        the reconstructed data by ART algorithm
+    """
+
+    # create xtomo_dataset object and initialize it
+    d = _generate_xtomo_object(data=data, white=white, dark=dark, theta=theta, center=center)
+
+    data_recon = d.art(iters=iters, overwrite=False)
+
+    return data_recon
+
+def save_data(data_recon, file_name, axis=0):
+    """Write reconstructed data to stack of tif files
+
+    Parameters
+    ----------
+    data_recon : np.ndarray
+        output reconstructed data
+
+    file_name : str
+        name of output file
+
+    axis : int, optional
+        the axis to read along the images
+    """
+
+    # Write to stack of TIFFs.
+    tomopy.xtomo_writer(data_recon, file_name, axis=axis)
+
+########################################
```

Comment:
Please remove dead code.  Version control will keep it around if we want it back.